_pages/SDK/v7.1/022-basics/0223-basics-measurementsandvalues.md

```diff
@@ -165,6 +165,7 @@ Property
 <nobr><code>DateTime?</code> FromModificationDate</nobr>                              | Specifies a date to select all measurements that where modified after that date.
 <nobr><code>DateTime?</code> ToModificationDate</nobr>                                | Specifies a date to select all measurements that where modified before that date.
 <nobr><code>int</code> LimitResult</nobr>                                             | The maximum number of measurements that should be returned, unlimited if set to -1 (default).
+<nobr><code>int</code> LimitResultPerPart </nobr>                                     | Restricts the number of result items per part. This parameter only takes effect in combination with either PartPath and Deep or PartUuids:<br> PartPath & Deep: trigger a deep search returning at max LimitResultPerPart measurements for each child part.<br> PartUuids: return at max LimitResultPerPart measurements for each specified part. Unlimited if set to -1 (default).
```

Comment:
I recommend to write "PartPath & Deep" to indicate those two belong together. Otherwise it is a bit harder to understand that "and" has a highter priority than "or".

Reply:
Good point, done.